### DISCUSSION TITLE
How To: Get the demos to run on Ubuntu 23.04

### DISCUSSION THREAD

**Discussion by MatthewBacon-Neteo** [bepu/bepuphysics2#274] [Show and tell]
Trival solution but didnt see it described else where so thought i would share here.

Cloned master branch

Problem: missing libs / freetype couldnt be found when trying to build the demo solution using Rider.

Freetype lib: I set the LD debug path as an environment variable as the debug error messages suggest but didnt get it to work but the alias below solved the problem.

aliased /usr/lib/x86_64-linux-gnu/libfreetype.so.6 to  /usr/lib/libfreetype.so.6

installed missing libs:

 sudo apt install -y libopentk-cil-dev libgbm-dev libegl-dev
 
 Rider can now run the project without any errors.
 
 Thanks to everyone that worked on this project, just discovered and looks awesome.

**Comment by RossNordby** [comment]
Thanks!

I don't maintain the demos side of things for other platforms, so I appreciate it when other people pick up the slack :P
